locales/en-US.json

```diff
@@ -142,6 +144,16 @@
   "people": "People",
   "person": "Person",
   "play": "Play",
+  "playback": {
+    "stopPlayback": "Stop playback",
+    "playbackSource": {
+      "item": "Playing from {item}",
+      "shuffle": "Playing in shuffle mode",
+      "shuffleItem": "Playing {item} in shuffle",
```

Comment:
"shuffle mode" to match previous line.

Reply:
Thank you for pointing that out.

Actually the latter is more recent than the former, so the former should be "Playing in shuffle".

The reasoning behind is that I think we should call "shuffle mode" what's enabled by the button on the player to shuffle the actual queue, while keeping "shuffle" term alone for starting new playbacks with shuffled tracks in the beginning.

Will change it tomorrow.

Reply:
A viable alternative for starting new playbacks could be "shuffle play"

Reply:
@hamburglar2160 I'm going to use that one for the button's tooltips 👍.